.env.sample

```diff
@@ -14,6 +14,7 @@ AMPLITUDE_API_KEY=API-KEY
 REDIS_URL=redis://127.0.0.1:6379
 SKYLIGHT_AUTHENTICATION=
 RUBYOPT='-W:no-deprecated -W:no-experimental'
+RECAPTCHA_SECRET_KEY_V3=
```

Comment:
@duranmla is this one different from the other we have?

Also, do we need RECAPTCHA_SITE_KEY anymore? or we can remove it?

Reply:
According to this https://developers.google.com/recaptcha/docs/faq I think that:

- If there is no other place using reCAPTCHA then we can remove what we used to have.
- We need a separate value key for the v3 reCAPTCHA